### PR TITLE
Add native method stubs for jdk.jfr.internal.jvm

### PR DESCRIPTION
**Pull request by thallium**

### CODE REVIEW

**Comment by thallium** [comment]
@tajila FYI

File: runtime/jcl/CMakeLists.txt [hunk 1]
```diff
@@ -238,6 +238,14 @@ if(NOT JAVA_SPEC_VERSION LESS 19)
 	)
 endif()
 
+# if(J9VM_OPT_JFR)
```
Comment (tajila) [inline runtime/jcl/CMakeLists.txt:241]: why are the ifdefs commented out?

Comment (thallium) [reply runtime/jcl/CMakeLists.txt]: my bad, I forgot it was commented out

File: runtime/jcl/uma/jfr_exports.xml [hunk 2]
```diff
@@ -21,4 +21,69 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 -->
 <exports group="jfr">
 	<export name="Java_com_ibm_oti_vm_VM_triggerExecutionSample" />
+	<export name="Java_jdk_jfr_internal_JVM_registerNatives" />
```
Comment (tajila) [inline runtime/jcl/uma/jfr_exports.xml:24]: youll also need a objects.xml for UMA build system

Comment (tajila) [reply runtime/jcl/uma/jfr_exports.xml:24]: then youll need to update runtime/jcl/module.xml to add both the objects and the exports file

Comment (thallium) [reply runtime/jcl/uma/jfr_exports.xml:24]: Sorry for the late reply, I've done the changes

File: runtime/jcl/module.xml [hunk 3]
```diff
@@ -65,6 +65,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 	<xi:include href="uma/se9_objects.xml"></xi:include>
 	<xi:include href="uma/se16_objects.xml"></xi:include>
 	<xi:include href="uma/se19_objects.xml"></xi:include>
+	<xi:include href="uma/jfr_objects.xml"></xi:include>
```
Comment (tajila) [inline runtime/jcl/module.xml:68]: you need an entry in the objects list below as well

```
			<group name="jfr">
				<include-if condition="spec.flags.opt_jfr"/>
			</group>
```

**Review by tajila — COMMENTED** [review]
Try building this in the internal farm with vmfarm on java8. Also try building it with openj9 with `--without-cmake` config flag

File: runtime/jcl/uma/jfr_objects.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] https://openjdk.org/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+<objects group="jfr">
+	<object name="Java_jdk_jfr_internal_JVM" >
```
Comment (tajila) [inline runtime/jcl/uma/jfr_objects.xml:23]: this needs to be `jdk_jfr_internal_JVM`

**Comment by tajila** [comment]
Were you ableto build internally on xa and wi?

**Comment by tajila** [comment]
jenkins test sanity xlinux jdk17

**Comment by tajila** [comment]
jenkins test sanity xlinux jdk17

**Pull request merged**